client/courses.html

```diff
@@ -48,14 +48,14 @@
   </ol>
   <!-- Example row of columns -->
   <div class="row">
-    <div class="col-xs-6 col-md-6">
+    <div class="col-md-6 col-sm-12 col-xs-12">
```

Comment:
It looks like you made some of the same changes here as in #34.  I think this may have been a mistake.  Maybe you left your text editor open while switching branches and didn't close/reopen files to refresh their contents?